subsys/bluetooth/audio/shell/bap.c

```diff
@@ -4061,6 +4171,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 #if defined(CONFIG_BT_BAP_BROADCAST_SINK)
 	SHELL_CMD_ARG(create_broadcast_sink, NULL, "0x<broadcast_id>", cmd_create_broadcast_sink, 2,
 		      0),
+	SHELL_CMD_ARG(create_by_broadcast_name, NULL, "0x<broadcast_name>",
```

Comment:
Please add an example of this in the `bap.rst` shell documentation file

Reply:
Ok, I'll add it.